.github/workflows/gcc.yml

```diff
@@ -56,28 +57,27 @@ jobs:
       # Cache spack, compiler and dependencies
       - name: cache-env
         id: cache-env
-        uses: actions/cache@v3
+        uses: actions/cache@v4
         with:
           path: |
             spack
             ~/.spack
-          key: spack-${{ runner.os }}-${{ env.cache_key }}-${{ hashFiles('gsi-utils/ci/spack.yaml') }}
+          key: spack-${{ runner.os }}-${{ env.cache_key }}-${{ hashFiles('gsi-utils/ci/spack_gcc.yaml') }}
 
       # Install dependencies using Spack
       - name: install-dependencies-with-spack
-        if: steps.cache-env.outputs.cache-hit != 'true'
+        # if: steps.cache-env.outputs.cache-hit != 'true'
```

Comment:
will this not cause the stack to be built each time for `gcc`?

Reply:
Yes, I think you are correct.  This will make the CI process quite long most of the time.  This should probably be uncommented (here and in the GSI).

Reply:
Indeed, gcc CI took 38 minutes.  I will uncomment the line in question and see what happens.